controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_webhook.go

```diff
@@ -50,6 +52,8 @@ func (in *KubeadmControlPlane) Default() {
 	if in.Spec.InfrastructureTemplate.Namespace == "" {
 		in.Spec.InfrastructureTemplate.Namespace = in.Namespace
 	}
+
+	in.Spec.Version = util.NormalizedKubernetesBuildVersion(in.Spec.Version)
```

Comment:
Do we actually want to change user input prefixing the `v` rather than allowing users to learn?

Reply:
Hah, fair point